lib/mustache-parser.pegjs

```diff
@@ -21,6 +22,14 @@ newMustache = name:newMustacheStart m_* attrs:(m_bracketedAttrs / newMustacheAtt
   return ret;
 }
 
+m_blockParams = m_blockStart m_* "|" m_* params:blockParamNames+ "|" {
+  return params;
+}
+
+m_blockStart = "as"
+
+blockParamNames = newMustacheAttrValue
```

Comment:
> That would also fall prey to the issue I mention above, letting through nonsense strings inside the pipes.

This seems OK though. The less handlebars/ember syntax Emblem's parser knows about, the more stable it is.